src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -392,6 +395,7 @@ export class SQLFuture extends Disposable implements FutureInternal {
 			if (maxRows && maxRows > 0) {
 				this.configuredMaxRows = maxRows;
 			}
+			this._stopOnError = config[SqlKernelStopOnErrorConfigName] ? config[SqlKernelStopOnErrorConfigName] : false;
```

Comment:
I am assuming you are trying to check whether it is undefined, if not, return the value otherwise default value is false. false or undefined both goes to the false after :.  this can be simplified to !!config[SqlKernalStopOnErrorConfigName]

Reply:
👍 please use `!!` syntax for simplicity.

Reply:
Or if you need to handle False string, maybe explictly check / handle that case. Only false (boolean), false/False (string) would be values that disable it.

Reply:
Thanks. Changed.